### DISCUSSION TITLE
Is there a Schema CRD to manage schemas and their evolution ?

### DISCUSSION THREAD

**Discussion by ee07dazn** [strimzi/strimzi-kafka-operator#6643] [Q&A] [answered]
I am trying to understand if there is a Schema CRD in strimzi operator to create and/or manage schemas in the schema registry? So far, from what i have looked/researched, I don't think there is atm but just want to double check in case if i have missed out on something ? An equivalent in confluent is https://docs.confluent.io/operator/current/co-manage-schemas.html

**Comment by scholzj** [comment, marked as answer]
Strimzi does not include any Schema Registry and does not manage schemas. You can pick any registry you want and use it. It is basically used by Kafka clients anyway.
